tests/hacker.test.js

```diff
@@ -603,6 +651,30 @@ describe("PATCH update one hacker", function () {
         });
     });
 
+    // hacker should fail to checkin hacker due to unconfirmed email
+    it("should FAIL to check in hacker as a volunteer due to unconfirmed email", function (done) {
```

Comment:
Hackers can be volunteers?

Reply:
no it means a volunteer tried to check in a hacker